### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#277] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152318381-53ce9541-497c-4207-8655-b2f1a4c6a8ca.png)

**Comment by GattoBrizzolato** [comment]
Non possiamo dimostrare che R sia una relazione di equivalenza in quanto le premesse non ci permettono di dimostrare la proprietà simmetrica di R : niente proprietà simmetrica niente equivalenza.

**Comment by Elia-Belli** [comment]
Dimostriamo che $R$ è riflessiva, simmetrica e transitiva:

- **Riflessività**: $R$ è riflessiva per ipotesi;

- **Simmetria**: la proprietà $aRb \land aRc \Rightarrow bRc$ vale $\forall a,b,c \in A$ ma non dice che $a,c$ debbano essere distinti: quindi se $aRb \land aRa \Rightarrow bRa$, siccome $aRa$ è sempre vero per la riflessività;

- **Transitività**: dimostrando la simmetria possiamo riscrivere le ipotesi della proprietà anche all'inverso (entrambe o solo una), ovvero $bRa \land cRa$ o $bRa \land aRc$ o ..., questo, per esempio, perchè se aRb è soddisfatta per simmetria è implicato esista anche bRa. Sfruttiamo quindi la simmetria per dimostrare la transitività: 

  Se $aRb \land aRc \Rightarrow bRc$, dobbiamo dimostrare che esiste anche aRc;
  Se $bRa \land aRc \Rightarrow aRc$, $bRa$ è garantito esistere per simmetria e così la transitività è dimostrata.

Le 3 proprietà sono soddisfatte quindi $R$ è un relazione d'equivalenza.
